src/plugins/home/public/application/components/guided_onboarding/getting_started.tsx

```diff
@@ -154,8 +155,11 @@ export const GettingStarted = () => {
     [guidedOnboardingService]
   );
 
+  // set up A/B testing
+  useVariation(cloudExperiments!, 'guided.onboarding', guideVersion, setGuideVersion);
```

Comment:
Suggest to make it like this: 

const guideVersion = useVariation<GuideVersion>(cloudExperiments, 'guided.onboarding', 'guide' /*default value here*/)


So this hook better matched the getVariation interface